test/integration/run_int_test.go

```diff
@@ -192,7 +192,6 @@ func (suite *RunIntegrationTestSuite) TestTwoInterrupts() {
 	suite.createProjectFile(ts, 3)
 
 	ts.LoginAsPersistentUser()
-	defer ts.LogoutUser()
```

Comment:
This seems reasonable to me. I don't understand the rationale for removing this.

Reply:
Each integration test runs with its own copy and configuration of the state tool meaning that each tests starts unauthenticated. I believe this is the only instance in our test suite where we defer a logout which is why it was removed.